packages/suite/src/components/suite/PrerequisitesGuide/DeviceUpdateRequired.tsx

```diff
@@ -19,8 +19,8 @@ export const DeviceUpdateRequired = () => {
         <TroubleshootingTips
             label={<Translation id="FW_CAPABILITY_UPDATE_REQUIRED" />}
             cta={
-                <Button onClick={handleClick}>
-                    <Translation id="TR_SEE_DETAILS" />
```

Comment:
Chaneg requested by @hynek-jina now the isn't like "See details" but directly "Install" in red (destructive) variant

Reply:
aha, so John changed it to make it completely "warning-like".

SO I'll keep his state